code/modules/mod/mod_theme.dm

```diff
@@ -737,17 +723,13 @@
 	armor = list(MELEE = 30, BULLET = 25, LASER = 25, ENERGY = 15, BOMB = 40, RAD = 25, FIRE = INFINITY, ACID = 150)
 
 /datum/mod_theme/magnate
-	name = "magnate"
-	desc = "A fancy, very protective suit for Nanotrasen's captains. Shock, fire and acid-proof while also having a large capacity and high speed."
-	extended_desc = "They say it costs four hundred thousand credits to run this MODsuit... for twelve seconds. \
-		The Magnate suit is designed for protection, comfort, and luxury for Nanotrasen Captains. \
-		The onboard air filters have been preprogrammed with an additional five hundred different fragrances that can \
-		be pumped into the helmet, all of highly-endangered flowers. A bespoke Tralex mechanical clock has been placed \
-		in the wrist, and the Magnate package comes with carbon-fibre cufflinks to wear underneath. \
-		My God, it even has a granite trim. The double-classified paint that's been painstakingly applied to the hull \
-		provides protection against shock, fire, and the strongest acids. Onboard systems employ meta-positronic learning \
-		and bluespace processing to allow for a wide array of onboard modules to be supported, and only the best actuators \
-		have been employed for speed. The resemblance to a Gorlex Marauder helmet is *purely* coincidental."
+	name = "'Magnate' command"
+	desc = "An advanced command modsuit developed by Nanotrasen for high-ranking corporate officers. Heavily armored, highly mobile, and plenty of space for modifications."
+	extended_desc = "An in-house design by Nanotrasen R&D, the Magnate encases the user inside a shell of plastitanum alloy with an ablative surface coating, giving robust protection against ballistic and energy-based threats in equal measure, and granting respectable protection from corrosive substances as well. \
+		An under-layer of ultra low-conductivity ceramic provides protection against the heat of raging fires and high intensity electrical discharges.\
+		The use of custom articulation first utilized in the NA-20 'Daedalus' means that the Magnate will not encumber the user under most circumstances, allowing them to move freely despite the mass of the suit. \
+		The over-engineered electrical and hydraulic systems also give it exceptional capacity for modification, allowing endless choice in how exactly a captain may wish to drain the resources of their robotics department. \
+		Overall, the Magnate provides protection rivalled only by the 'Jaeger' MODsuit of Syndicate infamy. Nanotrasen's public relations department continues to reiterate that any resemblance that parts of the suit may have to designs used by Gorlex are <b>purely</b> coincidental."
```

Comment:
"Nanotrasen's public relations department continues to reiterate that any resemblance that parts of the suit may have to designs used by Gorlex are <b>purely</b> coincidental." Is a bit of a mouthful, I would suggest lightening it, for instance : 

"Nanotrasen's PR Department reiterates that any resemblance with Gorlex's Jaeger design is <b>purely</b> coincidental."